src/dev-app/connected-overlay/connected-overlay-demo.ts

```diff
@@ -92,16 +99,19 @@ export class ConnectedOverlayDemo {
     if (this.overlayRef) {
       this.overlayRef.dispose();
       this.overlayRef = null;
-      this.showBoundingBox = false;
+      this.isBoundingBoxVisible = false;
     }
   }
 
-  toggleShowBoundingBox() {
+  showBoundingBox(showBoundingBox: boolean) {
     const box = document.querySelector<HTMLElement>('.cdk-overlay-connected-position-bounding-box');
 
     if (box) {
-      this.showBoundingBox = !this.showBoundingBox;
-      box.style.background = this.showBoundingBox ? 'rgb(255, 69, 0, 0.2)' : '';
+      if (showBoundingBox) {
+        box.classList.add('demo-bounding-box-visible');
+      } else {
+        box.classList.remove('demo-bounding-box-visible');
+      }
```

Comment:
Why has this been updated? I'm asking because I'd expect only changes which are relevant to Ivy / type checking.

Reply:
I ran into this while verifying the components via the dev-app pages. Previously if you clicked the checkbox, it would check itself and then the previous code here would instantly uncheck the box. Then it you toggled it off again the behavior would be the opposite of what was expected. I fixed it to verify that it wasn't an Ivy bug, which it wasn't, just a buggy demo.

Reply:
Sounds good. Just checking so that we don't oversee any potential breaking changes.